src/afpacket_plugin/afpacket_collector.cpp

```diff
@@ -324,7 +333,7 @@ void start_afpacket_collection(process_packet_pointer func_ptr) {
         for (int cpu = 0; cpu < num_cpus; cpu++) {
 
 // Well, we have thread attributes from Boost 1.50
-#if defined(BOOST_THREAD_PLATFORM_PTHREAD) && BOOST_VERSION / 100 % 1000 >= 50
+#if defined(BOOST_THREAD_PLATFORM_PTHREAD) && BOOST_VERSION / 100 % 1000 >= 50 && defined(__GLIBC__)
```

Comment:
If it does not work with musl it's reasonable to raise report to Boost. This binding code is extremely important for performance reasons.